paper-server/src/main/java/org/bukkit/craftbukkit/inventory/view/builder/CraftBlockEntityInventoryViewBuilder.java

```diff
@@ -44,23 +51,36 @@ protected AbstractContainerMenu buildContainer(final ServerPlayer player) {
             return buildFakeTile(player);
         }
 
+        trySetDefaultTitle(entity);
         return atBlock;
     }
 
     private AbstractContainerMenu buildFakeTile(final ServerPlayer player) {
-        if (this.builder == null) {
-            return handle.create(player.nextContainerCounter(), player.getInventory());
-        }
         final MenuProvider inventory = this.builder.build(this.position, this.block.defaultBlockState());
         if (inventory instanceof final BlockEntity tile) {
             tile.setLevel(this.world);
+            trySetDefaultTitle(tile);
+        }
+
+        if (!this.forceTile) { // gets around open noise for chest
```

Comment:
We can get a better name here I am sure.
Mostly because I guess all the other "tile" references in the code here were not caught in the initial review xD
`forceBlockEntityUsage`, `useBlockEntityMenuCreation`, whatever.
If you are at it, there are a couple of other `tile` names around, might be a vibe to update those to `blockEntity` too :)